testnet/stacks-node/src/config.rs

```diff
@@ -494,11 +496,97 @@ impl Config {
                     rbf_fee_increment: burnchain
                         .rbf_fee_increment
                         .unwrap_or(default_burnchain_config.rbf_fee_increment),
-                    epochs: match burnchain.epochs {
-                        Some(epochs) => Some(epochs),
-                        None => default_burnchain_config.epochs,
-                    },
+                    epochs: default_burnchain_config.epochs,
+                };
+
+                if let Some(ref conf_epochs) = burnchain.epochs {
```

Comment:
Could this block go inside a new function so that we get to keep the branch `Some(epochs) => Some(epochs)`?